openpype/tools/publisher/widgets/overview_widget.py

```diff
@@ -336,13 +337,16 @@ def _on_instance_context_change(self):
         self.instance_context_changed.emit()
 
     def _on_convert_requested(self):
-        _, _, convertor_identifiers = self.get_selected_items()
-        self._controller.trigger_convertor_items(convertor_identifiers)
+        self.convert_requested.emit()
 
     def get_selected_items(self):
         view = self._subset_views_layout.currentWidget()
         return view.get_selected_items()
 
+    def get_selected_convertors(self):
+        _, _, convertor_identifiers = self.get_selected_items()
+        return convertor_identifiers
```

Comment:
What's a convertor? What is being converted? Looking around the codebase this appears to be related to converting legacy instances but I can tell from just reading the code here I was clueless - this might need some decent docstrings or should maybe include the word `legacy`? Or are convertors not only intended for legacy?